examples/Demo/Shared/Components/SiteSettings.razor.cs

```diff
@@ -8,46 +8,45 @@ public partial class SiteSettings
 {
     private IDialogReference? _dialog;
     
-    [Inject]
-    private GlobalState GlobalState { get; set; } = default!;
+    //[Inject]
```

Comment:
Could you remove the unused code?

Reply:
Sure, I'll make sure to remove all the unused codes